### PR TITLE
[No ticket] Add more functionality in testing

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add local storage mocks to test happy path
- Add jwt decode mock to works decode jwt token functionality
- Change fetch request to axios token

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -22,11 +26,31 @@ const response = {
...
+};
+global.localStorage = localStorageMock;
+
 describe('App component', () => {
   afterEach(cleanup);
 
   beforeEach(() => {
-    axios.get = jest.fn(() => Promise.resolve(response));
+    axios.get = jest.fn((url) => {
+      if(url===(process.env.REACT_APP_API_URL + '/Reports/Reports/GetUserData')) {
```
Comment (cbernat) [inline src/App.test.js]: Could you fix the spacing between `===` and other symbols here?

File: src/App.test.js [hunk 2]
```diff
@@ -50,5 +83,10 @@ describe('App component', () => {
     const userEmail = getByText(response.data.user.Email);
 
     expect(userEmail).toBeDefined();
+    expect(axios.get).toHaveBeenCalledTimes(1);
+    expect(axios.get).toHaveBeenCalledWith(process.env.REACT_APP_API_URL + '/Reports/Reports/GetUserData', {
```
Comment (cbernat) [inline src/App.test.js]: Maybe this `process.env.REACT_APP_API_URL + '/Reports/Reports/GetUserData' ` can be defined in a variable to reuse.

File: src/App.test.js (hunk 1 above)
Comment (andresmoschini) [reply src/App.test.js]: Does not Prettify or JSLint detect it?

Comment (cbernat) [reply src/App.test.js]: Seems he did not run it I suspect...

File: src/App.test.js [hunk 3]
```diff
@@ -8,6 +8,10 @@ import App from './App';
 import { IntlProvider } from 'react-intl';
 import messages_es from './i18n/es.json';
 import messages_en from './i18n/en.json';
+import jwt_decode from 'jwt-decode';
+
+//Add mock to decode jwt token and not fail
```
Comment (andresmoschini) [inline src/App.test.js]: Very arguably, but I think that these comments are not necessary. What do you think?

Comment (fcoronelmakingsense) [reply src/App.test.js]: yes could be

File: src/App.test.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/App.test.js]: i forgot run. now i run and fixed

Comment (andresmoschini) [reply src/App.test.js]: There are Prettify plugins for most text editors, I do not like them, but maybe you do.

**Pull request merged**